stdlib/2/__builtin__.pyi

```diff
@@ -1315,7 +1315,12 @@ def oct(__i: int) -> str: ...  # TODO __index__
 
 if sys.version_info >= (3, 6):
     def open(file: Union[str, bytes, int, _PathLike], mode: str = ..., buffering: int = ..., encoding: Optional[str] = ...,
-             errors: Optional[str] = ..., newline: Optional[str] = ..., closefd: bool = ...) -> IO[Any]: ...
+             errors: Optional[str] = ..., newline: Optional[str] = ..., closefd: bool = ...,
+             opener: Optional[Callable[[str, int], int]] = ...) -> IO[Any]: ...
+elif sys.version_info >= (3, 3):
```

Comment:
Since we don't support Python versions < 3.4, this should be merged with the branch below. In fact, we removed all those not-required branches previously.